netstandard/ref/System.Xml.XPath.cs

```diff
@@ -213,6 +213,7 @@ public abstract partial class XPathNavigator : System.Xml.XPath.XPathItem, Syste
         public override object ValueAs(System.Type returnType, System.Xml.IXmlNamespaceResolver nsResolver) { throw null; }
         public virtual void WriteSubtree(System.Xml.XmlWriter writer) { }
     }
+    [System.Diagnostics.DebuggerDisplayAttribute("Position={CurrentPosition}, Current={debuggerDisplayProxy}")]
```

Comment:
Should not be exposed

Reply:
Agreed this is only needed in the implementation.